src/os/bluestore/BlueFS.cc

```diff
@@ -2216,6 +2223,15 @@ int BlueFS::open_for_read(
   return 0;
 }
 
+void BlueFS::_close_reader(FileReader *h)
+{
+  FileRef file = h->file;
+  delete h;
```

Comment:
May be assign h to nullptr and check for that on entry? Just for the sake of purity...

Reply:
not sure what you mean? i just mirrored the close_writer behavior.  h is passed by value to assigning it doesn't help

Reply:
yeah, my bad - missed it's not a member...